.vscode/settings.json

```diff
@@ -1,3 +1,18 @@
 {
-    "rust-analyzer.rustc.source": "discover"
+    "rust-analyzer.rustc.source": "discover",
+    "rust-analyzer.cargo.buildScripts.overrideCommand": [
+        "./x.py",
```

Comment:
Will this format work on Windows?

Reply:
Great point! I should be more mindful of these things.